man/Makefile.am

```diff
@@ -1,5 +1,11 @@
 AUTOMAKE_OPTIONS = gnu
 
```

Comment:
i am using to test
```
sphinx-build -b man -d doctrees -c . $PWD /tmp/man $PWD/8/ceph-syn.rst
```

and see http://sphinx-doc.org/builders.html#sphinx.builders.manpage.ManualPageBuilder , i am not sure that it is able to act as a translator of a single file. maybe i can have something like `conf.py.in` but that's a little bit overkill, IMHO.